demo/b2share_demo/cli.py

```diff
@@ -236,6 +237,50 @@ def is_same_url(url1, url2):
 
 @demo.command()
 @with_appcontext
+@click.option('-u', '--update', is_flag=True, default=False)
+@click.argument('base_url')
+def swap_pids(update, base_url):
+    """ Fix the invalid creation of new ePIC_PIDs for migrated files. Swaps with the old b2share v1 PID that we stored in alternate_identifiers and puts the wrongly created ePIC_PID in alternate_identifiers. Note this creates a new version of the invenio record (at the time of writing we do not show the latest version of invenio record objects)
+    """
+    record_search = requests.get(urljoin(base_url, "api/records"),
+                                 {'size': 1000, 'page': 1},
+                                 verify=False)
+    records = record_search.json()['hits']['hits']
+    for rec in records:
+        inv_record = Record.get_record(rec['id'])
+        aids = None
+        if 'alternate_identifiers' in inv_record.keys():
+            aids = inv_record['alternate_identifiers']
+            found = False
+            for aid in aids:
+                if aid['alternate_identifier_type']=='ePIC_PID':
+                    new_pid = aid['alternate_identifier']
+                    _pid = inv_record['_pid']
+                    for pid in _pid:
+                        if pid['type']=='ePIC_PID':
+                            old_pid = pid['value']
+                            found = True
```

Comment:
after this line there should be two 'break' statement: it's possible there are multiple PIDs, but only the first instance should be considered